### DISCUSSION TITLE
Porting dependant critical libraries to version 3

### DISCUSSION THREAD

**Discussion by waynevanson** [gcanti/fp-ts#1530] [General]
There are many libraries which work with and are dependent on fp-ts.

I'm concerned about integrating them to v3.
- Which libraries are critical for a porting, if any?
- In the documentation, would we have a section that advises which one's are on version 2 and version 3?

I really want to use v3 but I rely heavily on external libraries that used v2 as it's dependency.

**Comment by samhh** [comment]
In my view the following are critical:

- fp-ts-contrib
- io-ts
- io-ts-types
